src/UnitTransformedLikelihood.cc

```diff
@@ -43,7 +43,7 @@ typedef std::vector< shapelet::MatrixBuilderFactory<Pixel> > FactoryVector;
  * Function intended for use with std algorithms to compute the cumulative sum
  * of the number of pixels in a sequence of EpochFootprints
  */
-int componentPixelSum(int partialNumPixels, CONST_PTR(EpochFootprint) const &epochImagePtr) {
+int componentPixelSum(int partialNumPixels, std::shared_ptr<const EpochFootprint> const &epochImagePtr) {
```

Comment:
[DM style](https://developer.lsst.io/cpp/style.html#the-const-keyword-should-be-listed-after-the-type-name) is to write 
```suggestion
int componentPixelSum(int partialNumPixels, std::shared_ptr<EpochFootprint const> const &epochImagePtr) {
```